X/Xyce/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Xyce"
+version = v"7.2.0"
+
+# Collection of sources required to complete build
+sources = [
+            GitSource("https://github.com/Xyce/Xyce.git", "a61faef4bfb2f36f1aa7cc44264bbbb66fbaac11"),
+            DirectorySource("./bundled")
+          ]
+
+# Bash recipe for building across all platforms
+script = raw"""
+export TMPDIR=${WORKSPACE}/tmpdir
+mkdir ${TMPDIR}
+
+cd $WORKSPACE/srcdir
+apk add flex-dev
+install_license ${WORKSPACE}/srcdir/Xyce/COPYING
+cd Xyce
+if [[ $target != i686-linux-gnu ]] && [[ $target != x86_64-linux-gnu ]] ; then
+        atomic_patch -p1 ${WORKSPACE}/srcdir/patches/cross.patch
+fi
+./bootstrap
+cd ..
+mkdir buildx
+cd buildx
+/workspace/srcdir/Xyce/./configure --enable-shared --disable-mpi --prefix=${prefix} LDFLAGS="-L${libdir} -lopenblas" CPPFLAGS="-I/${includedir} -I/usr/include" --host=${target}
+make -j${nprocs}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+   
+platforms= [Platform("x86_64", "linux", libc="glibc", cxxstring_abi="cxx11", libgfortran_version=v"4.0.0")]
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libxyce", :libxyce),
+    LibraryProduct("libNeuronModels", :libNeuronModels),
+    LibraryProduct("libADMS", :libADMS),
+    ExecutableProduct("Xyce", :Xyce)
```

Comment:
@ViralBShah do I just export the 
```
LibraryProduct("libxyce", :libxyce),
ExecutableProduct("Xyce", :Xyce)
```

(Similar to `ngspice`)

Reply:
Yes just those will be sufficient.